src/controllers/resize-controller.js

```diff
@@ -33,19 +33,25 @@ export class ResizeController {
     host.addController(this);
 
     /** @type {number|null} the latest observed `width` of the host */
-    this.width = null;
+    this._width = null;
 
     this._host = host;
 
-    /** @type {Function|undefined} if defined, the callback is executed everytime a resize happens */
     this._callback = options?.callback;
 
-    /** @type {number[]|undefined} if defined sets attributes `w-lt-${breakpoint}` & `w-gte-${breakpoint}` for each breakpoint */
     this._widthBreakpoints = options?.widthBreakpoints;
 
     this._observer = new ResizeObserver(() => this._onResize());
   }
 
+  get width () {
+    return this._width;
+  }
+
+  set width (newWidth) {
```

Comment:
fix: Oh no, you don't want anybody but the controller itself to change the width. This is the whole point of making width private with only a getter on it. (This makes the width readonly from outside this class)

Reply:
ahhh yes of course :sweat_smile:  thanks